### PR TITLE
 Schedule crash aggregate view dataset with Airflow

### PR DESCRIPTION
**Pull request by vitillo**
This change is [Reviewable](https://reviewable.io/reviews/vitillo/telemetry-airflow/10)

### CODE REVIEW

File: dags/crash_aggregates.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from airflow import DAG
+from datetime import datetime, timedelta
+from operators.emr_spark_operator import EMRSparkOperator
+
+default_args = {
+    'owner': 'mdoglio@mozilla.com',
+    'depends_on_past': True,
+    'start_date': datetime(2016, 6, 27),
```
Comment (maurodoglio) [inline dags/crash_aggregates.py]: This should be now()-timedelta(days=1)

Comment (vitillo) [inline dags/crash_aggregates.py]: I don't think so; that's the start date of the dag, i.e. the first time the dag should ever be scheduled.

File: dags/crash_aggregates.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    'retries': 3,
+    'retry_delay': timedelta(minutes=30),
+}
+
+dag = DAG('crash_aggregates', default_args=default_args, schedule_interval='@daily')
+
+t0 = EMRSparkOperator(task_id = "crash_aggregate_view",
+                      job_name = "Crash Aggregate View",
+                      instance_count = 9,
+                      env = {"date": "{{ ds_nodash }}", "bucket": "{{ task.__class__.airflow_bucket }}"},
```
Comment (maurodoglio) [inline dags/crash_aggregates.py]: what is ds_nodash?

File: dags/crash_aggregates.py (hunk 1 above)
Comment (maurodoglio) [inline dags/crash_aggregates.py]: Ah, I guess I need to read some docs 😛

**Comment by maurodoglio** [comment]
Thank you ! :shipit:

**Pull request merged**